client/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "name-request",
-  "version": "0.1.5",
+  "version": "0.1.6",
```

Comment:
FYI I think this is good practise as an easy way to know what's deployed (at least, once we implement the footer info tooltip).